commands/issue.go

```diff
@@ -213,12 +217,23 @@ hub-pr(1), hub(1)
 		--color
 `,
 	}
+
+	cmdAddComment = &Command{
+		Key: "comment",
+		Run: addComment,
+		KnownFlags: `
+    -m, --message MSG
+    -f, --file
```

Comment:
This is necessary to indicate to the args parser that `-f` and `--file` flags expect a value:
```suggestion
    -f, --file FILE
```